File: miraculous-frontend/miraculous-ui/src/dto/like-dto/like-dto.ts
export class LikeDto {

  id: number;
  likeGiverId: number;
  dateOfLike: string;


  constructor(id?: number, likeGiverId?: number, dateOfLike?: string) {
    this.id = id;
    this.likeGiverId = likeGiverId;
    this.dateOfLike = dateOfLike;
  }
}

File: miraculous-backend/src/main/java/com/example/miraculousbackend/dto/LikeDto.java
package com.example.miraculousbackend.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class LikeDto {

    private Long id;
    private Long likeGiverId;
    private String dateOfLike;
}
